io.github.swordpuffin.wardrobe.json

```diff
@@ -0,0 +1,42 @@
+{
+    "id" : "io.github.swordpuffin.wardrobe",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "47",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "wardrobe",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--talk-name=org.freedesktop.Flatpak",
```

Comment:
there is no point in flatpaking something if every command requires spawning flatpak-spawn https://github.com/SwordPuffin/Wardrobe/blob/845c2667715850a856078e9fd139b68c83d354f0/src/window.py#L59-L63

Reply:
also some of them are exposed inside the sandbox through settings portal

Reply:
there's a portal to set the wallpaper too https://flatpak.github.io/xdg-desktop-portal/docs/doc-org.freedesktop.portal.Wallpaper.html